bootstrap.js

```diff
@@ -73,6 +73,17 @@ function setDefaultPrefs() {
       break;
     }
   }
+  
+  //set new tabs to open at top as default -- only if undefined
+  try {
+    Services.prefs.getBoolPref('extensions.verticaltabs.opentabstop');
+  } catch (ex) {
+    if (ex.result === Components.results.NS_ERROR_UNEXPECTED) {
+      Services.prefs.setBoolPref('extensions.verticaltabs.opentabstop', true);
```

Comment:
Okay, one last thing…
I think we might want to remove this in removeDefaultPrefs, too.